src/main/target/SITL/target.c

```diff
@@ -181,12 +205,55 @@ static void* udpThread(void* data) {
     while (workerRunning) {
         n = udpRecv(&stateLink, &fdmPkt, sizeof(fdm_packet), 100);
         if (n == sizeof(fdm_packet)) {
-//            printf("[data]new fdm %d\n", n);
+            if (!fdm_received) {
+                printf("[SITL] new fdm %d t:%f from %s:%d\n", n, fdmPkt.timestamp, inet_ntoa(stateLink.recv.sin_addr), stateLink.recv.sin_port);
+                fdm_received = true;
+                // pwmLink.si.sin_addr = stateLink.recv.sin_addr;
+            }
             updateState(&fdmPkt);
         }
     }
 
-    printf("udpThread end!!\n");
+    printf("[SITL] udpThread end!!\n");
+    return NULL;
+}
+
+static float readRCSITL(const rxRuntimeState_t *rxRuntimeState, uint8_t channel) {
+    UNUSED(rxRuntimeState);
+    return rcPkt.channels[channel];
+}
+
+static uint8_t rxRCFrameStatus(rxRuntimeState_t *rxRuntimeState)
+{
+    UNUSED(rxRuntimeState);
+    return RX_FRAME_COMPLETE;
+}
+
+
+static void* udpRCThread(void* data) {
+    UNUSED(data);
+    int n = 0;
+
+    while (workerRunning) {
+        n = udpRecv(&rcLink, &rcPkt, sizeof(rc_packet), 100);
+        if (n == sizeof(rc_packet)) {
+            if (!rc_received) {
+                printf("[SITL] new rc %d: t:%f AETR: %d %d %d %d AUX1-4: %d %d %d %d\n", n, rcPkt.timestamp,
+                    rcPkt.channels[0], rcPkt.channels[1],rcPkt.channels[2],rcPkt.channels[3],
+                    rcPkt.channels[4], rcPkt.channels[5],rcPkt.channels[6],rcPkt.channels[7]);
+                
+                rxRuntimeState.channelCount = SIMULATOR_MAX_RC_CHANNELS;
```

Comment:
This online takeover of receiver is quite suspicious. It's OK for SITL in current state, but eventually it shall be cleaned up, possibly by adding UDP as one of serial receivers ... 

For now, can you add comment about current code flow?